src/layouts/Container.js

```diff
@@ -8,10 +8,9 @@ import Footer from '../components/Footer'
 import Loading from '../components/Loading'
 
 const Main = styled.main`
-    padding: 5.5rem 0;
-    min-height: 90vh;
```

Comment:
this line is important, it ensures that the page always fills the screen

Reply:
👍